alexandria/core/tests/test_api.py

```diff
@@ -200,7 +200,7 @@ def test_api_patch(fixture, admin_client, viewset, snapshot, minio_mock):
     assert_response(response, context, snapshot, request_payload=data)
 
 
-@pytest.mark.freeze_time("2017-05-21")
+@pytest.mark.freeze_time("2020-05-21")
```

Comment:
I don't see how this would contribute in solving flaky tests. Anyways something seems terribly off with the snapshots. Did you update the snapshots after you changed the seed?

Reply:
I didn't really believe it'd fix it either. I did update the snapshots. Still experimenting how to fix in CI - still works on my machine :tm:

Reply:
I think it was an outdated image. Updating it on my machine solved the issue